tests/conftest.py

```diff
@@ -134,7 +134,7 @@ def _print_config():
 def _check_docker_requirement():
     logger.debug("Checking if docker is running...")
     if settings.DISTRO == 'openSUSE_k8s' and \
-            settings.as_bool('BUILD_ROOK_FROM_GIT'):
+            converter('@bool', settings.UPSTREAM_ROOK.BUILD_ROOK_FROM_GIT):
```

Comment:
Why not `settings.as_bool('UPSTREAM_ROOK.BUILD_ROOK_FROM_GIT')` ?

Reply:
The as_bool method fails when using nested settings. I've just opened an upstream bug here:
https://github.com/rochacbruno/dynaconf/issues/445